avm/res/desktop-virtualization/scaling-plan/main.bicep

```
@@ -0,0 +1,227 @@
+metadata name = 'Azure Virtual Desktop Scaling Plan'
+metadata description = 'This module deploys an Azure Virtual Desktop Scaling Plan.'
+metadata owner = 'Azure/module-maintainers'
+
+@sys.description('Required. Name of the scaling plan.')
+param name string
+
+@sys.description('Optional. Location of the scaling plan. Defaults to resource group location.')
+param location string = resourceGroup().location
+
+@sys.description('Optional. Friendly name of the scaling plan.')
+param friendlyName string = name
+
+@sys.description('Optional. Time zone of the scaling plan. Defaults to UTC.')
+param timeZone string = 'UTC'
+
+@sys.description('Optional. Host pool type of the scaling plan.')
+param hostPoolType string = 'Pooled'
+
+@sys.description('Optional. Exclusion tag to be used for exclusion of VMs from scaling plan.')
+param exclusionTag string = ''
+
+@sys.description('Optional. Schedules of the scaling plan.')
+param schedules array = []
+
+@sys.description('Optional. Host pool references of the scaling plan.')
+param hostPoolReferences array = []
+
+@sys.description('Optional. Description of the scaling plan.')
+param description string = name
+
+@sys.description('Optional. Tags of the scaling plan.')
+param tags object?
+
+@sys.description('Optional. Array of role assignment objects that contain the \'roleDefinitionIdOrName\' and \'principalId\' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: \'/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11\'.')
```

```suggestion
@sys.description('Optional. Array of role assignments to create.')
```
Suggestion as per [specs](https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/).

---

Note: Requires regeneration of both the readme & json